apps/synapse-portal-framework/src/pages/FileEntityPage/FileEntityPage.tsx

```diff
@@ -30,42 +32,76 @@ function FileEntityPage() {
     { id: 'provenance', title: 'Provenance' },
   ].filter(Boolean) as DetailsPageSectionLayoutType[]
 
+  const title = entityBundle?.entity.name ?? ''
+  const type = entityBundle?.entityType ?? 'Entity'
+  const doiUri = entityBundle?.doiAssociation?.doiUri
+
   if (isLoading) {
     return (
       <Stack sx={{ my: 3 }}>
         <SynapseSpinner size={40} />
       </Stack>
     )
   }
+
+  const icon = (
+    <Box
+      sx={{
+        backgroundColor: 'rgba(0, 0, 0, 0.08)',
+        borderRadius: '50%',
+        width: '72px',
+        height: '72px',
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+      }}
+    >
+      <IconSvg
+        icon={'file'}
+        wrap={false}
+        sx={{ width: '32px', height: '32px' }}
+      />
+    </Box>
+  )
+
   return (
-    <Container>
-      <DetailsPageLayout>
-        <DetailsPageMenu menuSections={fileEntityPageSections} />
-        <Stack
-          sx={{
-            flex: 1,
-            gap: '40px',
-          }}
-        >
-          {entityBundle?.rootWikiId && (
-            <CollapsibleSection title="Wiki" id="wiki">
-              <MarkdownSynapse
-                ownerId={entityId}
-                wikiId={entityBundle?.rootWikiId}
+    <article>
+      <HeaderCard
+        type={type}
+        title={title}
+        description={''}
+        icon={icon}
+        doiUri={doiUri}
+      />
+      <Container>
+        <DetailsPageLayout>
+          <DetailsPageMenu menuSections={fileEntityPageSections} />
```

Comment:
Taking a look at these again since your last refactor, and I think it might be slightly cleaner to use the `DetailsPageContent` component and pass it a list of sections. It will render the menu and section headers for you, and ensure the style is more consistent with existing details pages.